### PR TITLE
OPBOX-8616 | refactor of component publications

### PR DESCRIPTION
**Pull request by rafixer**

### CODE REVIEW

**Review by pawel-piecyk — DISMISSED** [review]

File: upload-and-publish/action.yml [hunk 1]
```diff
@@ -17,60 +17,23 @@ inputs:
...
-        branchName: ${{ steps.extract_branch.outputs.branch }}
-
-    - name: Create tag
-      shell: bash
-      run: $GITHUB_ACTION_PATH/create-tag.sh
-      env:
-        branchName: ${{ steps.extract_branch.outputs.branch }}
-        gitUserEmail: ${{ inputs.gitUserEmail }}
+    - name: Upload
+      uses: allegro-actions/opbox-release-tools/upload@v1-faas-dev
```
Comment (KubaZ) [inline upload-and-publish/action.yml:27]: update version before merge

File: upload-and-publish/action.yml [hunk 2]
```diff
@@ -17,60 +17,23 @@ inputs:
...
+      with:
+        accessKeyId: ${{ inputs.accessKeyId }}
+        secretAccessKey: ${{ inputs.secretAccessKey }}
+        assets-domain: ${{ inputs.assets-domain }}
+
+    - name: Publish to npm
+      uses: allegro-actions/opbox-release-tools/publish/npm@v1-faas-dev
```
Comment (KubaZ) [inline upload-and-publish/action.yml:34]: update version before merge

File: upload-and-publish/action.yml (hunk 1 above)
Comment (rafixer) [reply upload-and-publish/action.yml]: done ;)

File: upload-and-publish/action.yml (hunk 2 above)
Comment (rafixer) [reply upload-and-publish/action.yml]: done ;)

**Review by KubaZ — APPROVED** [review]

**Review by bgalek — APPROVED** [review]

**Review by pawel-piecyk — APPROVED** [review]

**Pull request merged**